docs/agents/sematext-agent/containers/installation.md

```diff
@@ -0,0 +1,105 @@
+To deploy the Sematext Agent as a container, run the following command on each host:
+
+```bash
+docker run -d  --restart always --privileged -P --name st-agent \
+-v /sys/kernel/debug:/sys/kernel/debug \
+-v /var/run/:/var/run/ \
+-v /proc:/host/proc:ro \
+-v /etc:/host/etc:ro \
+-v /sys:/host/sys:ro \
+-v /usr/lib:/host/usr/lib:ro \
+-e CONTAINER_TOKEN==<YOUR_DOCKER_APP_TOKEN_HERE> \
+-e INFRA_TOKEN=<YOUR_INFRA_APP_TOKEN_HERE> \
+-e REGION=US \
```

Comment:
How will one know that EU is another valid option?  One would have to _read the text_ below.  Would _\<US or EU\>_ make more sense here?